### PR TITLE
[DS-3283] 6x Mirage2: Edit Collection Source - No Field Label for Set Id

### PR DESCRIPTION
**Pull request by terrywbrady**
Tested locally for Mirage and Mirage2 themes.

https://jira.duraspace.org/browse/DS-3283

## Before Change
image

## After Change
image

### CODE REVIEW

**Review by tdonohue — DISMISSED** [review]
@terrywbrady : this is currently failing compilation (according to Travis CI)

**Comment by terrywbrady** [comment]
@tdonohue , PR fixed.  Since I pulled this from master brance PR, I re-tested this on a local DSpace 6 instance.

File: dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java [hunk 1]
```diff
@@ -185,19 +185,13 @@ public void addBody(Body body) throws WingException, SQLException, AuthorizeExce
...
+            Radio oaiSetSettingRadio = settings.addItem().addRadio("oai-set-setting");
+            oaiSetSettingRadio.addOption("all".equals(oaiSetIdValue) || oaiSetIdValue == null, "all", "All sets");
+            oaiSetSettingRadio.addOption(!"all".equals(oaiSetIdValue) && oaiSetIdValue != null, "specific", "Specific sets");
 
-        Text oaiSetId = oaiSetComp.addText("oai_setid");
-	    oaiSetId.setSize(40);
-        if(!"all".equals(oaiSetIdValue) && oaiSetIdValue != null)
-        {
-            oaiSetId.setValue(oaiSetIdValue);
-        }
```
Comment (tdonohue) [inline dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java]: It's unclear to me why this `if` statement and `setValue()` were removed. Are these covered elsewhere? Or are they just not needed for some reason?

File: dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java [hunk 2]
```diff
@@ -185,19 +185,13 @@ public void addBody(Body body) throws WingException, SQLException, AuthorizeExce
...
-	    oaiSetId.setHelp(T_help_oaisetid);
```
Comment (tdonohue) [inline dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java]: Also unclear why the `setHelp()` is being removed.

File: dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java (hunk 1 above)
Comment (terrywbrady) [inline dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java]: In my hurry to recreate this PR, it looks like I merged changes incorrectly.  Thanks for catching this.

**Comment by terrywbrady** [comment]
Unsuccessful rebase... closing PR

**Pull request closed without merging**